src/helpers/s3.ts

```diff
@@ -19,11 +22,14 @@ async function upload(
 ): Promise<void> {
   await remove(bucket, key);
   const body = JSON.stringify(payload);
-  await s3.upload({ Bucket: bucket, Key: key, Body: body }).promise();
+  await new Upload({
+    client: s3,
+    params: { Bucket: bucket, Key: key, Body: body },
+  }).done();
 }
 
 async function remove(bucket: string, key: string) {
-  await s3.deleteObject({ Bucket: bucket, Key: key }, () => {}).promise();
+  await s3.deleteObject({ Bucket: bucket, Key: key }, () => {});
 }
 
 const S3 = { upload };
```

Comment:
This line seems to cause problem because we now have double definition for `S3`.

Reply:
Switched to export list in https://github.com/vimcolorschemes/vimcolorschemes/pull/899/commits/4ef17fbed41dfd74e25eeefd67a0df1666032d19